go.mod

```diff
@@ -11,14 +11,16 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.15.1
 	go.uber.org/zap v1.24.0
-	k8s.io/api v0.26.1
+	k8s.io/api v0.26.3
 	k8s.io/apiextensions-apiserver v0.26.1
-	k8s.io/apimachinery v0.26.1
-	k8s.io/client-go v0.26.1
-	k8s.io/utils v0.0.0-20230202215443-34013725500c // latest
-	sigs.k8s.io/controller-runtime v0.14.4
+	k8s.io/apimachinery v0.26.3
+	k8s.io/client-go v0.26.3
+	k8s.io/utils v0.0.0-20230313181309-38a27ef9d749 // latest
+	sigs.k8s.io/controller-runtime v0.14.5
 )
 
+require github.com/medik8s/common v1.0.0
```

Comment:
NIT: Why adding this require here and not grouped it with the other packages from above (lines 6-19)?

Reply:
I see it was modified in the [last commit](https://github.com/medik8s/node-healthcheck-operator/pull/212/commits/954af23f72c709a14111a250b69db0085400d0dc)....